scRNA-seq-advanced/03-dataset_integration.Rmd

```diff
@@ -373,21 +534,27 @@ scater::plotReducedDim(merged_sce,
                        # Some styling to help us see the points:
                        point_size = 0.5,
                        point_alpha = 0.2) +
-  ggplot2::ggtitle("UMAPs calculated on merged data")
+  # Modify the legend key so its points are larger and easier to see
+  guides(colour = guide_legend(override.aes = list(size = 3, alpha = 1))) +
+  # Add a plot title
+  ggtitle("UMAP calculated on merged_sce")
 ```
-The main "clustering" (not technical clustering, just what we see visually!) in these plots are the batches. 
-A well-integrated dataset will instead show much more mixing among batches, as the batch effects will have hopefully been corrected.
-How might you expect an integrated UMAP to look?
-
 
+Now that we've properly accounted for batches in the `merged_sce` object, we see that the libraries are separated in space, which more reasonably reflects that this data is _not yet batch-corrected_.
+Indeed, the primary "clustering" (not technical clustering, just what we see visually!) in these plots are the batches.
```

Comment:
Just noting batches/samples... whatever decision you make here on terminology...
```suggestion
Indeed, the primary "clustering" (not technical clustering, just what we see visually!) in these plots are the samples.
```